app/templates/application.hbs

```diff
@@ -1,19 +1,21 @@
-<div style="height: 350px;">
-  <div class="d-flex align-items-center flex-column justify-content-center h-100 shadow p-3 mb-5 bg-white rounded">
-    <div><img class="pull-left img-thumbnail " src="images/firm_initials.png">
-      <span class="display-5 font-weight-bold text-muted " >TAX DUE DATE SEARCH</span>
+{{#if checkRoute}}
```

Comment:
be sure to review my comments on the controller on how to do this using native controller methods. you will probably still need a computed property like this here though.